ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/link-tracking-groups.yml

```diff
@@ -0,0 +1,6 @@
+---
+link_tracking_groups:
+  - group_name: EVPN_MH_ES1
```

Comment:
Remember to update to `name`. Current molecule artifacts are empty because the name is not there.

Reply:
Sorry, I missed this.